src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +20,14 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(string -> string.split(","))
+                .flatMap(Arrays::stream)
+                .mapToInt(Integer::parseInt)
+                .filter(integer -> integer % 2 == 0)
+                .sorted()
+                .findFirst()
```

Comment:
```suggestion
                .min()
```
You can use min() instead